_docs/backenders/strategy-2021/roles/hiring.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Hiring role definition -- *draft*
```

Comment:
we should remove the "draft" and name the responsible person.

Reply:
I think nobody wanted to take this role and somehow it ended up being weighted as lower priority.

As Agnieszka commented in the PR thread, we should either close this PR and shelve it, or keep it as draft with no responsible person. In practice I think I have been taking care (in a very lightweight fashion) of some of these tasks and in general I don't mind keep bringing up the "let's march on with hiring" mantra all the time, and except for some revisions to the job ad text the level of effort is normally negligible, so I'd even be happy to keep bugging the team with "let's march on with hiring" 🤷🏼